packages/segment/package-lock.json

```diff
@@ -0,0 +1,5373 @@
+{
```

Comment:
Also rerun npm install after nvm use here

Reply:
switched to v15.2.1 (version that is in the `.nvmrc` and reinstalled all of the npm modules